tests/server/server.spec.ts

```diff
@@ -1,21 +1,38 @@
 import Server from '../../src/server/server'
-
 let hasMockClientConnected = false
 
 jest.mock('net', () => {
   type MaybeNumber = number | undefined
   type MaybeString = string | undefined
   type MaybeVoidFunction = (() => void) | undefined
```

Comment:
This can be changed in the next PR. There's a built in VoidFunction type. That can be used instead of `(() => void)`

Reply:
Got it! I'll be sure to keep that in mind for the next one. :)